parsers/US_AK_SAP.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime, timezone
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Session
+
+from electricitymap.contrib.lib.models.event_lists import ProductionBreakdownList
+from electricitymap.contrib.lib.models.events import ProductionMix
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.lib.exceptions import ParserException
+
+PARSER_NAME = "US_AK_SAP.py"
+
+SOURCE = "seapahydro.org"
+DATA_URL = "https://seapahydro.org/api/scada/index"
+
+
+def get_value(data: dict, key: str) -> float:
+    return float(data[key]["text"])
+
+
+def fetch_production(
+    zone_key: ZoneKey,
+    session: Session = Session(),
```

Comment:
It has previusly been pointed out to us/me that having a class constructor as the default argument can cause problems (but shouldn't).
I suggesst setting this to:
```suggestion
    session: Optional[Session] = None,
```
And then do
`session = session or Session()` inside the function call.